transport/src/androidUnitTest/kotlin/transport/core/AttachmentHandlerImplTest.kt

```diff
@@ -293,12 +293,14 @@ internal class AttachmentHandlerImplTest {
 
     @Test
     fun `when detach() on not processed attachment`() {
-        val result = subject.detach(TestValues.Token, "not processed attachment id")
+        val invalidAttachmentId = "not processed attachment id"
```

Comment:
- Rename invalidAttachmentId to givenAttachmentId to indicate that this is an input value we use for testing.
- Use TestValues.DEFAULT_STRING instead of hardcoded  "not processed attachment id".
- I would like to see a unit test in MCAttachmentTests ->  `when detach() non existing attachmentId` updated to reflect the change. Initial assumption of this test was that when detach non existing attachment attachmentHandler will return null. But now, it throws an exception.